src/map/script.c

```diff
@@ -20967,6 +20970,38 @@ BUILDIN_FUNC(npcshopupdate) {
 	return SCRIPT_CMD_SUCCESS;
 }
 
+// Clan System
+BUILDIN_FUNC(clan_join){
+	struct map_session_data *sd;
+	int clan_id;
+
+	clan_id = script_getnum(st,2);
+
+	if( !script_charid2sd( 3, sd ) ){
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	if( clan_member_join( sd, clan_id, sd->status.account_id, sd->status.char_id ) ){
+		return SCRIPT_CMD_SUCCESS;
+	}else{
+		return SCRIPT_CMD_FAILURE;
+	}
+}
+
+BUILDIN_FUNC(clan_leave){
+	struct map_session_data *sd;
+
+	if( !script_charid2sd( 2, sd ) ){
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	if( clan_member_leave( sd, sd->status.clan_id, sd->status.account_id, sd->status.char_id ) ){
+		return SCRIPT_CMD_SUCCESS;
+	}else{
+		return SCRIPT_CMD_FAILURE;
+	}
+}
```

Comment:
shouldn't the `else` part, just let it be end of function control?
(and for script above)